Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM golang:1.15-alpine AS cloud-build
+FROM golang:1.16-alpine AS cloud-build
```

Comment:
The build is now failing. Is the change from Daniel in plgd branch? Are we will using plgd branch or are we on master? If we are on master, let's wait until it's get there.

Reply:
Sometimes onboard/disown via sdk fails - it will be fixed by upgrade sdk in PR https://github.com/plgd-dev/cloud/pull/268